OCPBUGS-86569:  Update PTP Operator web console installation procedure (#112263)
openshift/openshift-docs: modules/nw-ptp-installing-operator-web-console.adoc

Reviewer: ```suggestion
If the installation later succeeds with an *Succeeded* message, you can ignore the *Failed* message.
```
To match the change in line 36, I guess we need to update this one too for consistency.

.. In the {product-title} web console, click *Ecosystem* -> *Software Catalog*.

.. Type `ptp` in the *Filter by keyword* box to find the PTP Operator.

.. Click the *PTP Operator* tile, and then click *Install*.

.. On the *Install Operator* page, ensure that *A specific namespace on the cluster* is selected and the *Operator recommended Namespace* `openshift-ptp` is shown. Click *Install*.

> .. The web console displays the install progress. When the installation is complete, click *View installed Operators*.

Reviewer: ```suggestion
.. Wait for the installation to complete and then click *View installed Operators*.
```
To keep imperative instructions.
